.taskcluster.yml

```diff
@@ -141,6 +145,13 @@ tasks:
                   expires: {$fromNow: '2 weeks'}
                   path: /bot.tar
                   type: file
+            routes:
+              - "index.project.code-review.bot.rev.${head_rev}"
+              - $if: 'tasks_for == "github-pull-request"'
+                then: "index.project.code-review.bot.pull_request.${head_branch}"
```

Comment:
Do we need this pull_request index?

Reply:
Could be useful to test an image on taskcluster before it's merged, as we won't have a dockerhub push anymore (i did that on fuzzing-tc, it's useful)